workflow-controller/controller-service/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.main.lazy.initialization=true
```

Comment:
Why again in application properties ?

Reply:
As per spring documentation it was required in the properties file